SwrveSDK/src/amazon/java/com/swrve/sdk/adm/SwrveAdmMessageReceiver.java

```diff
@@ -0,0 +1,10 @@
+package com.swrve.sdk.adm;
+
+import com.amazon.device.messaging.ADMMessageReceiver;
+
+//SwrveMessageReceiver listens for messages from ADM
```

Comment:
Such a strange implementation by amazon. Do we define this receiver in the androidmanifest.xml?

That example implementation is different. Its a static inner class.